modules/blocks/utils.js

```diff
@@ -370,11 +370,12 @@ Utils.prototype.aggregateBlocksReward = function (filter, cb) {
 	params.delegates = constants.activeDelegates;
 
 	if (filter.start !== undefined) {
-		params.start = filter.start - constants.epochTime.getTime () / 1000;
+		params.start = (filter.start - constants.epochTime.getTime()) / 1000;
```

Comment:
Are you sure?
- `filter.start` is UNIX timestamp or Lisk epoch?
- `constants.epochTime.getTime()` will return UNIX timestamp
- `params.start` needs to be block timestamp, so Lisk epoch time